runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -59,7 +59,7 @@
 import io.aklivity.zilla.runtime.engine.internal.stream.NamespacedId;
 import io.aklivity.zilla.runtime.engine.vault.Vault;
 
-public final class Engine implements AutoCloseable
+public final class Engine extends EngineBuilder implements AutoCloseable
```

Comment:
Please revert this change as `Engine` and `EngineBuilder` are different concepts.